airflow/models.py

```diff
@@ -69,6 +69,7 @@
 )
 from sqlalchemy.ext.declarative import declarative_base, declared_attr
 from sqlalchemy.orm import reconstructor, relationship, synonym
+import sqlalchemy.types as types
```

Comment:
nit: move this up with other sqlalchemy imports on line 65? don't think it needs an alias.